### PR TITLE
fix ALTER DATABASE character set processing

### PR DESCRIPTION
**Pull request by osheroff**
@zendesk/rules

### CODE REVIEW

File: src/test/java/com/zendesk/maxwell/schema/ddl/DDLIntegrationTest.java [hunk 1]
```diff
@@ -78,7 +78,7 @@ public void testAlter() throws SQLException, SchemaSyncError, IOException, Inter
 
 	@Test
 	public void testAlterDatabase() throws Exception {
-		String sql[] = {
+	String sql[] = {
```
Comment (vanchi-zendesk) [inline src/test/java/com/zendesk/maxwell/schema/ddl/DDLIntegrationTest.java]: indentation

**Comment by vanchi-zendesk** [comment]
No tests for where the failure is?

**Comment by osheroff** [comment]
@vanchi-zendesk adding the `diff` method caused some existing test to fail which I then  fixed.

**Comment by vanchi-zendesk** [comment]
:+1:

**Pull request merged**